projects/baby_driver/scripts/can_message.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# coding: utf-8
+"""python can message with 8 bytes"""
+from __future__ import print_function
+
+import can
+
+def send_message(msg):
+    """sends can message with 8 bytes. You can change the 2nd one"""
+    #change the channel later to can0 or vcan0
+    bus = can.interface.Bus(bustype='socketcan', channel='can0', bitrate=250000)
+
+    msg = can.Message(
+        arbitration_id=0xC0FFEE, data=[0, msg, 0, 0, 0, 0, 0, 0], is_extended_id=False
```

Comment:
arbitration ID is the device ID, this should at least be a constant and probably shouldn't be 0xC0FFEE but who knows. Also if we're going to use this generally you should definitely be able to change all the bytes not just the 2nd one!

Reply:
Yup, it was just an id. I can change it to like 0x2. Yeah, I didn't complete it because I wasn't sure what the format would be.
Like which byte would represent what. (Ex: byte 1 would be the function)
But I'll edit it to just do msg1, msg2, msg3.... and we can change the names later if necessary

Reply:
I think this method is just going to be a generic low-level "send a CAN message" function, so the data would just be a list of bytes and we'll give it an interpretation somewhere else.

Also, there is a native data type in python for an array of bytes: `bytearray`. I think instead of msg1, msg2, ... you can take a list called `msg`, convert it to a `bytearray`, and check that its length is <=8.

Reply:
Yup. Actually it can be a byte array or a single number. If the byte array is >8 then the can library just won't send it